README.md

```diff
@@ -1,85 +1,116 @@
 
-	For Mac
+## How to Contribute
 
-		brew install sphinx-doc
+You'll need to know how git works, and the syntax of either restructuredText or Markdown.
 
-		export PATH="/usr/local/opt/sphinx-doc/bin:$PATH"
+For option 2, you'll need to [install Sphinx and extensions](#install-sphinx-and-extensions), or [MkDocs and extensions](#install-mkdocs-and-extensions), or have [Docker](https://docs.docker.com/get-docker/) installed to preview any changes. (Of these, Docker is by far the simplest.)
 
-	Check the installation:
+There are three ways to get changes made to the documentation. Two are 'do it yourself', one is 'ask someone to do it'.
 
-		sphinx-build --version
-		sphinx-build 2.2.
+### Option 1: 'Do it yourself': Edit via Github
 
-2.	Make a fork of the pmm-doc repository, then make git clone of your repository locally.
+1. Each page of [PMM2 documentation](https://www.percona.com/doc/percona-monitoring-and-management/2.x/index.html) has a link to the `.rst`. and `.md` versions of the page.
 
-3.	Run the documentation build. The command to prepare html version is the following one:
-		
-		make html
+2. Click any link to be taken to the github edit page.
 
-	**Note:** if you are on BSD-based systems, you may need to comment on the line `@sed -i 's/{{ toc }}/{ toctree\(false\)"` in Makefile.
+3. Make your changes and commit. Unless you are a member of the Percona team, you'll be asked to fork the repository.
 
-	Check result:
+4. Do so and make a pull request for merging your changes.
 
-		copying static files... ... done
-		copying extra files... done
-		dumping search index in English (code: en)... done
-		dumping object inventory... done
-		build succeeded, 1319 warnings.
+### Option 2: 'Do it yourself': Edit a cloned copy
 
-		The HTML pages are in build/html.
+1. Fork and clone this repository.
 
-		Build finished. The HTML pages are in build/html.
+2. Make your changes in either the Sphinx/rst version (under `pmm-doc/source`) or MkDocs/md (under `pmm-doc/docs`).
 
-	You can see a lot of Warnings. This is normal.
+3. For all but the simplest changes, [preview the documentation](#preview-the-documentation).
 
-4.	Now compiled documentation is located in the `/build/html/` folder.
-	
-	You can simply open "/build/html/index.html" in your browser. 
+4. Commit and push the changes.
 
-	Or use Docker
+5. Make a pull request to merge the changes.
 
-		docker run -dit --name my-apache-app -p 8080:80 -v "$PWD"/build/html/:/usr/local/apache2/htdocs/ httpd:2.4
+### Option 3: 'Ask someone': Create a ticket
 
-5.	Check the documentation. It's going to be built without make-up because it's going to use percona.com's make-up.
+1. Create a ticket in our [Jira](https://jira.percona.com/projects/PMM/issues) system.
 
-	![Result](/images/img-readme-result.png)
+2. Describe the problem or improvement needed in as much detail as possible.
 
+3. You can communicate with developers and writers and you'll be notifed of progress as it progresses.
```

Comment:
hmmm..."progress as it progresses" sounds odd...maybe just "notified as it progresses"?